lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -117,15 +127,8 @@ class Accessibility extends FRGatherer {
       deps: [
         axeLibSource,
         pageFunctions.getNodeDetailsString,
+        createAxeRuleResultArtifact,
       ],
-    }).then(returnedValue => {
-      if (!returnedValue) {
-        throw new Error('No axe-core results returned');
-      }
-      if (!Array.isArray(returnedValue.violations)) {
```

Comment:
we trust our `evaluate()` by now and don't do these checks in other gatherers. No need to keep these

Reply:
ya